### PR TITLE
Display annotations on Alerts page

### LINKED ISSUE #1219: Alerts page should display expanded ANNOTATIONS
Even though the expansion happens in Prometheus, it is only _visible_ in Alertmanager. It should be available on `/alerts`.

### PR DESCRIPTION
**Pull request by roidelapluie**
Fixes #1219

Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

**Comment by roidelapluie** [comment]
anno

**Comment by brian-brazil** [comment]
Thanks, these clutter up the page quite a bit. Can you hide them by default or something?

**Comment by roidelapluie** [comment]
They are hidden by default, with a checkbox on the top.

**Comment by brian-brazil** [comment]
It's not really "Labels and Annotations" if the annotations aren't there by default. Maybe have them as another column that's hidden by default?

**Comment by roidelapluie** [comment]
That will not fit a screen width. What about changing the title of the column of it is enabled?

**Comment by roidelapluie** [comment]
Updated

**Comment by roidelapluie** [comment]
anno
It looks better now

File: web/ui/templates/alerts.html [hunk 1]
```diff
@@ -38,6 +43,21 @@ <h2>Alerts</h2>
               <td>{{.ActiveAt.UTC}}</td>
               <td>{{.Value}}</td>
             </tr>
+                {{ if .Annotations.Map}}
+                  <tr style="display:none" class="alert_annotations">
```
Comment (brian-brazil) [inline web/ui/templates/alerts.html]: The <tr should be aligned with the others

Comment (roidelapluie) [reply web/ui/templates/alerts.html]: I have aligned the code

**Comment by simonpasquier** [comment]
:+1:

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**